core/modules/wmgr/python/dbMgr.py

```diff
@@ -232,12 +240,15 @@ def dropDb(dbName):
 
     # use non-privileged account
     dbConn = Config.instance().dbConn()
-    if not dbConn.dbExists(dbName):
+    if not dbExists(dbConn, dbName):
         raise ExceptionResponse(404, "DatabaseMissing", "Database %s does not exist" % dbName)
     try:
-        dbConn.dropDb(dbName)
+        dropDb_(dbConn, dbName)
     except db.DbException as exc:
-        _log.error('exception when dropping database %s: %s', dbName, exc)
+        _log.error('Db exception when dropping database %s: %s', dbName, exc)
+        raise
+    except sqlalchemyExc.OperationalError as exc:
```

Comment:
Hmm, I think that `mysqlUtils` is not supposed to leak any sqlalchemy exceptions.